### DISCUSSION TITLE
When comparing time periods, include comparisons for pages, sources, regions et al

### DISCUSSION THREAD

**Discussion by gravelld** [plausible/analytics#3394] [Anything Else]
When comparing two time periods, the topmost portlet includes a comparison of the two dates:

![Screenshot_2023-10-04_10-23-31](https://github.com/plausible/analytics/assets/873704/f95b0f81-6333-4486-8460-1402cfef0a89)

However, the Top Sources, Pages and other portlets below have no such comparison.

It's possible to drill into individual pages, sources etc so that the topmost portlet reloads with a comparison of the two dates. But it would be really useful to see comparisons for all pages, sources etc together. Otherwise there's a lot of click and back, click and back.

This would be really useful for comparing two time periods. Overall visit stats and individual stats are useful, but seeing an overview helps understand what you should drill into.

**Comment by siaminos** [comment]
Yes, please!

I want to be able to compare two dates and see page-level differences in pages views as well as the percentage difference. This was basic functionality in Universal Analytics that I used contantly. Basically, seeing how specific pages compare over time.
